hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/support/translators/ConsensusSubmitMessageTranslator.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.services.bdd.junit.support.translators;
+
+import static com.hedera.hapi.node.base.ResponseCodeEnum.SUCCESS;
+
+import com.hedera.hapi.block.stream.output.StateChange;
+import com.hedera.hapi.block.stream.output.StateChanges;
+import com.hedera.hapi.node.transaction.TransactionReceipt;
+import com.hedera.hapi.node.transaction.TransactionRecord;
+import com.hedera.node.app.state.SingleTransactionRecord;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import java.util.List;
+import org.jetbrains.annotations.NotNull;
+
+public class ConsensusSubmitMessageTranslator implements TransactionRecordTranslator<SingleTransactionBlockItems> {
+    private static final long RUNNING_HASH_VERSION = 3L;
```

Comment:
It would be nice to leave a comment here pointing to the protobuf, so that anyone reading the code will know why this is a constant

Reply:
This link will take one directly to the explanation:
https://github.com/hashgraph/hedera-protobufs/blob/pbj-storage-spec-review/block/stream/output/consensus_service.proto#L6

Reply:
Added.